cmd/cli/main.go

```diff
@@ -32,10 +32,10 @@ func main() {
 	var rootCmd = &cobra.Command{
 		Use: "mainflux-cli",
 		PersistentPreRun: func(cmd *cobra.Command, args []string) {
-			cli.ParseConfig()
+			cliConf:= cli.ParseConfig(sdkConf)
 
-			sdkConf.MsgContentType = sdk.ContentType(msgContentType)
-			s := sdk.NewSDK(sdkConf)
+			cliConf.MsgContentType = sdk.ContentType(msgContentType)
+			s := sdk.NewSDK(cliConf)
```

Comment:
This will not allow to set different content type via toml, So Apply the content type only if `MsgContentType` is empty

Reply:
OKay